mayastor/src/grpc/v1/pool.rs

```diff
@@ -0,0 +1,324 @@
+                        {
+                            return Err(LvsError::Invalid {
+                                source: Errno::EINVAL,
+                                msg: format!(
+                                    "invalid uuid {}, found pool with uuid {}",
+                                    args.uuid.unwrap(),
+                                    pool.uuid(),
+                                ),
+                            });
+                        }
+                        pool.destroy().await?;
+                    } else {
+                        return Err(LvsError::Invalid {
+                            source: Errno::EINVAL,
+                            msg: format!("pool {} not found", args.name,),
+                        });
+                    }
+                    Ok(())
+                })?;
+
+                rx.await
+                    .map_err(|_| Status::cancelled("cancelled"))?
+                    .map_err(Status::from)
+                    .map(Response::new)
+            },
+        )
+        .await
+    }
+
+    #[named]
+    async fn export_pool(
+        &self,
+        request: Request<ExportPoolRequest>,
+    ) -> GrpcResult<()> {
+        self.locked(
+            GrpcClientContext::new(&request, function_name!()),
+            async move {
+                let args = request.into_inner();
+                info!("{:?}", args);
+                let rx = rpc_submit::<_, _, LvsError>(async move {
+                    if let Some(pool) = Lvs::lookup(&args.name) {
+                        if args.uuid.is_some() && args.uuid != Some(pool.uuid())
+                        {
+                            return Err(LvsError::Invalid {
+                                source: Errno::EINVAL,
+                                msg: format!(
+                                    "invalid uuid {}, found pool with uuid {}",
+                                    args.uuid.unwrap(),
+                                    pool.uuid(),
+                                ),
+                            });
+                        }
+                        pool.export().await?;
+                    } else {
+                        return Err(LvsError::Invalid {
+                            source: Errno::EINVAL,
+                            msg: format!("pool {} not found", args.name,),
+                        });
+                    }
+                    Ok(())
+                })?;
+
+                rx.await
+                    .map_err(|_| Status::cancelled("cancelled"))?
+                    .map_err(Status::from)
+                    .map(Response::new)
+            },
+        )
+        .await
+    }
+
+    #[named]
+    async fn import_pool(
+        &self,
+        request: Request<ImportPoolRequest>,
+    ) -> GrpcResult<Pool> {
+        self.locked(
+            GrpcClientContext::new(&request, function_name!()),
+            async move {
+                let args = request.into_inner();
```

Comment:
are we missing an info! here?

Reply:
Added `info!("{:?}", args);` to all the rpc functions as it looked much cleaner

Reply:
Do we also need an info! for the success case at the end?

Reply:
There is a success message in all the called functions.